tools/cpp/CROSSTOOL.tpl

```diff
@@ -990,12 +978,140 @@ toolchain {
       action: 'c-compile'
       action: 'c++-compile'
       flag_group {
-        flag: "/O2"
+        flag: "/O2" # Implies /Og /Oi /Ot /Oy /Ob2 /Gs /GF /Gy
         flag: "/DNDEBUG"
       }
     }
   }
 
+  feature {
+    name: 'treat_warnings_as_errors'
```

Comment:
I think this warrants a "disabling" feature (e.g. some teams might want to enable the feature globally in bazelrc, and then opt out individual targets.